tests/system_tests_http2_tls.py

```diff
@@ -711,3 +711,103 @@ def test_zzz_http_connector_delete(self):
+        cls.http_listener_props = {'port': cls.tester.get_port(),
+                                   'address': 'examples',
+                                   'host': 'localhost',
+                                   'name': cls.listener_name,
+                                   'protocolVersion': 'HTTP2',
+                                   'authenticatePeer': 'yes',
+                                   'sslProfile': 'http-listener-ssl-profile'}
+        config_qdra = Qdrouterd.Config([
+            ('router', {'mode': 'interior', 'id': 'QDR.A'}),
+            ('listener', {'port': cls.tester.get_port(), 'role': 'normal', 'host': '0.0.0.0'}),
+            # curl will connect to this httpListener and run the tests.
+            ('httpListener', cls.http_listener_props),
+            ('sslProfile', {'name': 'http-listener-ssl-profile',
+                            'caCertFile': cls.ssl_file('ca-certificate.pem'),
+                            'certFile': cls.ssl_file('server-certificate.pem'),
+                            'privateKeyFile': cls.ssl_file('server-private-key.pem'),
+                            'password': 'server-password'}),
+            ('listener', {'role': 'inter-router', 'port': inter_router_port})
+        ])
+
+        cls.connector_name = 'connectorToBeDeleted'
+        cls.connector_props = {
+            'port': cls.nginx_port,
+            'address': 'examples',
+            'host': 'localhost',
+            'protocolVersion': 'HTTP2',
+            'name': cls.connector_name,
+            # Verifies host name. The host name in the certificate sent by the server must match 'localhost'
+            'verifyHostname': 'yes',
+            'sslProfile': 'http-connector-ssl-profile'
+        }
+        config_qdrb = Qdrouterd.Config([
+            ('router', {'mode': 'interior', 'id': 'QDR.B'}),
+            ('listener', {'port': cls.tester.get_port(), 'role': 'normal', 'host': '0.0.0.0'}),
+            ('httpConnector', cls.connector_props),
+            ('connector', {'name': 'connectorToA', 'role': 'inter-router',
+                           'port': inter_router_port,
+                           'verifyHostname': 'no'}),
+            ('sslProfile', {'name': 'http-connector-ssl-profile',
+                            'caCertFile': cls.ssl_file('ca-certificate.pem'),
+                            'certFile': cls.ssl_file('client-certificate.pem'),
+                            'privateKeyFile': cls.ssl_file('client-private-key.pem'),
+                            'password': 'client-password'}),
+        ])
+        cls.router_qdra = cls.tester.qdrouterd("http2-two-router-tls-A", config_qdra, wait=True)
+        cls.router_qdrb = cls.tester.qdrouterd("http2-two-router-tls-B", config_qdrb)
+        cls.router_qdra.wait_router_connected('QDR.B')
+        cls.router_qdrb.wait_router_connected('QDR.A')
+        cls.router_qdrb.wait_http_server_connected(is_tls=True)
+        wait_http_listeners_up(cls.router_qdra.addresses[0])
```

Comment:
this feels like a huge amount of setup code given that the test can be summarized with one line of comment; just a thought, I am not sure what to do about it, and there's probably nothing to do about it right now